apps/src/executionLog.js

```diff
@@ -24,9 +24,16 @@ module.exports.getResultsFromLog = function (logConditions, executionLog) {
      * @param {!Object} condition
      * @param {string} [condition.matchType] 'exact', 'inexact'
      * @param {number} [condition.minTimes] number of matches required
+     * @param {number} [condition.maxTimes] maximum number of matches allowed
      * @param {string[]} [condition.entries] function or statement names
      * @param {string} [condition.message] message to display if condition fails
      */
+
+    condition.minTimes = condition.minTimes || 0;
+    if (typeof condition.maxTimes === 'undefined') {
+      condition.maxTimes = Infinity;
+    }
```

Comment:
yeah, I tried, but it made the nice simple mocha test code get a lot more complicated because utils pulls in hammer, which requires a `window` object, which made me decide to ditch using utils :)